src/Kernels.cpp

```diff
@@ -1,6 +1,5 @@
 #include <cmath>
```

Comment:
I notice that we don't use `std::` math functions only ones in the global namespace.  We should either change the header or use the `std::` namespace explicitly.

Reply:
Agreed. I think it should be part of a separate PR, though.

Reply:
Agreed.